Cargo.toml

```diff
@@ -1,35 +1,6 @@
-[package]
-name = "sia_core"
-version = "0.0.1"
-edition = "2021"
-repository = "https://github.com/SiaFoundation/core-rs"
-license = "MIT"
-description = "Low-level SDK for interacting with the Sia decentralized storage network"
-authors = ["The Sia Foundation"]
-categories = ["cryptography::cryptocurrencies"]
-keywords = ["sia", "decentralized", "blockchain", "depin", "storage"]
-
-[lib]
-name = "sia_core"
-path = "src/lib.rs"
-
-[dependencies]
-base64 = "0.22.1"
-bip39 = "2.1.0"
-blake2b_simd = "1.0.2"
-ed25519-dalek = "2.1.1"
-hex = "0.4.3"
-rayon = "1.10.0"
-serde = { version = "1.0.214", features = ["derive"] }
-serde-big-array = "0.5.1"
-serde_json = "1.0.132"
-sha2 = "0.10.8"
-thiserror = "1.0.67"
-
-[dev-dependencies]
-rand = "0.8.5"
-criterion = { version = "0.5" }
-
-[[bench]]
-name = "merkle_root"
-harness = false
+[workspace]
+resolver = "2"
+members = [ 
+	"sia",
+	"sia_sdk_derive",
```

Comment:
I thought using nightly we can get rid of the two crates?

Reply:
We can, but I don’t want to force nightly to build